web/pgadmin/browser/static/js/node_view.jsx

```diff
@@ -210,6 +210,7 @@ export function getNodeView(nodeType, treeNodeInfo, actionType, itemNodeData, fo
         key={itemNodeData?._id}
         formType={formType}
         getInitData={initData}
+        updatedData={{rows_cnt: itemNodeData?.rows_cnt}}
```

Comment:
I think this should be conditional and for now, this data should be passed only for table, view and places where count row is applicable.